product_packaging_multi_barcode/views/product_template_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_template_form_view_multi_barcode" model="ir.ui.view">
+        <field name="name">product.template.form.multi.barcode.inherited</field>
+        <field name="model">product.template</field>
+        <field
+            name="inherit_id"
+            ref="product_multi_barcode.product_template_form_view_multi_barcode"
+        />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='barcode_ids']//field[@name='name']"
+                position="after"
+            >
+                <field name="packaging_id" />
+                <field name="create_date" />
```

Comment:
Our client needs to know the create date of the barecode. Not sure if it can be something generic. So I think we could provide that as an option `optional="hide"`

Reply:
Changed